### PR TITLE
rust-vcprompt 0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by vjt**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Adds rust-vcprompt 0.1.0 from https://github.com/sscherfke/rust-vcprompt by @sscherfke.

Renames the binary to `rust-vcprompt` to avoid coexistence with other similar tools.

### CODE REVIEW

File: Formula/rust-vcprompt.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  homepage "https://github.com/sscherfke/rust-vcprompt"
+  url "https://github.com/sscherfke/rust-vcprompt/archive/0.1.0.tar.gz"
+  sha256 "3c51261c7b75f04de566d97a95530521f71fb8b99529deab5ce8af97702c7acf"
+  head "https://github.com/sscherfke/rust-vcprompt.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--release"
+    mv "target/release/vcprompt", "target/release/rust-vcprompt"
```
Comment (DomT4) [inline Formula/rust-vcprompt.rb]: You can merge this into the below, like so:
```ruby
bin.install "target/release/rust-vcprompt" => "rust-vcprompt"
```

File: Formula/rust-vcprompt.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    bin.install "target/release/rust-vcprompt"
+  end
+
+  test do
+    mkdir "repo"
```
Comment (DomT4) [inline Formula/rust-vcprompt.rb]: Similar thing here. `mkdir "repo" do` will replace the two-step `mkdir` and `cd`.

**Comment by DomT4** [comment]
> rust-vcprompt:
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)

This is likely to be a significant issue, given it's not even close.

**Comment by vjt** [comment]
Changes done. About ⭐️ ⭐️ ⭐️ ... I've added mine. The tool is worthwhile so I am sure it'll gain more in the future. In the meantime, this formula is available on [my tap](https://github.com/vjt/homebrew-formulas).

Thanks!

**Comment by sscherfke** [comment]
I could also integrate your formula directly into my repo.

**Comment by vjt** [comment]
@sscherfke as you prefer, I can maintain it in my tap and deliver new releases once you tag them, or you can do it :-) no problem for me, I just wanted a reproducible way to have this installed on my box. :-)

**Comment by ilovezfs** [comment]
We'll be happy to reconsider this in the future when it meets the notability requirements. Thanks for the PR @vjt!

**Comment by sscherfke** [comment]
I created an “official” tap for rust-vcprompt: https://github.com/sscherfke/homebrew-rust-vcprompt. Since it’s not in the core, the executable remains to be `vcprompt`.  The original vcprompt project is marked as conflicting.

**Pull request closed without merging**